anitya/tests/test_config.py

```diff
@@ -52,7 +52,7 @@
 
 librariesio_platform_whitelist = ['pypi', 'rubygems']
 
-default_regex = '%(name)s(?:[-_]?(?:minsrc|src|source))?[-_]([^-/_\s]+?)(?i)(?:[-_](?:minsrc|src|source|asc|release))?\.(?:tar|t[bglx]z|tbz2|zip)'
```

Comment:
You could also just split the string over multiple lines, as you prefer :)

Reply:
I already fixed this by using dummy regex in the test.
Your solution will not work (tested) `pytoml` has issue with reading multiline string.